### PR TITLE
Copy figure 3c into final directory

### PR DESCRIPTION
**Pull request by sjspielman**
Per Issue #1253, this PR copies the mutational signatures sina/IQR plot into the final location in the `figures/` directory. No plotting code itself is changed, only the `generate_figures.sh` script.

### CODE REVIEW

File: figures/generate-figures.sh [hunk 1]
```diff
@@ -143,3 +143,11 @@ fi
 Rscript -e "rmarkdown::render('${analyses_dir}/cnv-chrom-plot/cn_status_heatmap.Rmd',
                               clean = TRUE, params = list(final_figure=TRUE))"
 
+
+
+######## Mutational signatures
+# Copy the figure to final directory
+cp ${analyses_dir}/mutational-signatures/plots/cns/exposures_sina_IQR.pdf  pdfs/fig3/panels/mutational_signatures_exposuures.pdf
```
Comment (jharenza) [inline figures/generate-figures.sh:150]:
```suggestion
cp ${analyses_dir}/mutational-signatures/plots/cns/exposures_sina_IQR.pdf  pdfs/fig3/panels/mutational_signatures_exposures.pdf
```

**Review by jharenza — APPROVED** [review]
just a quick typo fix and looks good!

**Review by jaclyn-taroni — APPROVED** [review]
My only comment, and I think it's a future thought, is maybe we should reorganize this shell script a bit so everything in the same figure is grouped. 🤷🏻

**Comment by sjspielman** [comment]
> My only comment, and I think it's a future thought, is maybe we should reorganize this shell script a bit so everything in the same figure is grouped.

I also thought this. I'll put in an issue.

**Pull request merged**